apidoc/Titanium/UI/Picker.yml

```diff
@@ -353,10 +353,9 @@ properties:
         If `true`, selection indicator is enabled.
 
         On iOS 7 and later, the picker indicator is always shown and you cannot control it.
-
-        On iOS 6 and prior, if enabled, a blue bar is displayed to indicate the current selection.
+        Manually setting the property was deprecated in 6.1.0 and will be ignored by the SDK.
```

Comment:
It was written in the iOS-paragraph, so I hoped people will get it's iOS related, but I'll explicitly say it :-).